implementations/typescript/ockam/ockam_app/tailwind.config.cjs

```diff
@@ -2,5 +2,13 @@
 export default {
   content: ['./src/**/*.{html,js,svelte,ts}'],
   plugins: [],
+  theme: {
+    fontSize: {
+      sm: ['11px'],
```

Comment:
I would recommend to generally use `ems`/`rems` units rather than pixels for font sizing.

https://www.w3docs.com/snippets/css/which-is-better-to-use-in-css-px-em-or-rem.html

The upstream font utility classes use `rems`:
https://tailwindcss.com/docs/font-size

I believe most or all of this change would be undone if we decide to stick to `rems`?

Reply:
I've done the conversion to `rem`. 

> I believe most or all of this change would be undone if we decide to stick to rems?

What do you mean by this? Changing the root font size directly?

Reply:
The content here in `tailwind.config.cjs`, if rewritten in terms of `rems`, might be duplicating the original definitions, unless the specific numbers differ. For the ones that are identical to the originals, we could omit them here.

Reply:
I've updated the `tailwind.config.cjs` file with rem values.